package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "mattermost-webapp",
```

Comment:
I'm always confused on this change, do we need that or we should remove it?

Reply:
I think we need that. The node version I am on right now adds it automatically, but the one I was earlier was removing it.